pkg/runtime/conversion_generator.go

```diff
@@ -529,6 +588,14 @@ func (g *conversionGenerator) writeFooter(b *buffer, indent int) {
 	b.addLine("}\n", indent)
 }
 
+func (g *conversionGenerator) conversionFunctionCall(inType, outType reflect.Type, scopeName string, args ...string) string {
```

Comment:
I like this approach - thanks!

Since this is replacing usage of "existsDedicatedConversionFunction" - can you please replace the only remaining call to it and remove it?